client/src/payloadDisplay.tsx

```diff
@@ -49,9 +54,17 @@ export const PayloadDisplay = ({
             "application/vnd.mediaservice.kahuna.image",
             "true"
           );
+          sendTelemetryEvent?.(PINBOARD_TELEMETRY_TYPE.DRAG_FROM_PINBOARD, {
+            assetType: payloadAndType?.type,
+            tab: tab as Tab,
```

Comment:
this looks ugly, and there's possibly type problems from doing this - tried looking for a better option and unfortunately the best I could find was

```suggestion
            ...(tab && { tab }),
```

which is not elegant but should ensure no problems with sending an undefined through